index.html

```diff
@@ -191,23 +192,20 @@ <h3>Student Projects</h3>
 
     <div class="container marketing" id="contact">
         <div class="list-head col-xs-7 col-sm-7 col-md-7 contact-left">
-            For any queries related to Science and Technology Council (SNTC) or any of its clubs, please contact the Technical Secretary at the given details.
+            <p>For any queries related to Science and Technology Council (SNTC) or any of its clubs, please contact the Technical Secretary at the given details.</p>
```

Comment:
@divyashk sys_admins.htm ke link ko neeche hi rehne de. Sirf Technical Secretary ki details edit karde bas

Reply:
The sys-admin link is placed outside the "page", so it can been seen in a weird position in every "page".It needs fixing.
If you want  then I can position it more towards the bottom.(can't leave it as it is though)

Reply:
@divyashk It seems intentional to me, had it been a bug it would have been issues list. Plus I feel that the concept of reaching out to maintainers easily from any tab seems good to me. If you see webpages of any organization or technical society, you will see the contact details in footer of the page and that is what this is similar to.